test/functional/shared.js

```diff
@@ -176,9 +181,26 @@ function invalidAssets (account, option, badTransactions) {
 	});
 }
 
+function multisigScenario (size, amount) {
+	this.account = node.randomAccount();
```

Comment:
this.account and this.transaction are never used externally - they can be private (`var`)

Reply:
They are used.

Reply:
multisigScenario is a constructor function, should begin with capital letter.